src/components/Modal/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import MaterialModal from '@material-ui/core/Modal';
+import Backdrop from '@material-ui/core/Backdrop';
+import { createStyles, makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    modal: {
+      position: 'fixed',
+      top: '50%',
+      left: '50%',
+      transform: `translate(-50%, -50%)`,
+    },
+  }),
+);
+
+interface ModalProps {
+  open: boolean;
+  onClose: (event: object, reason: string) => void;
+  children: any;
```

Comment:
The type of [ModalProps.children](https://github.com/mui-org/material-ui/blob/next/packages/material-ui/src/Modal/Modal.d.ts#L20) is `React.ReateElement`, is there a reason for specifying the type in a wider range(`any`)?

Reply:
Thank you for your review
Do you mean `ReactElement`??
At first, I wanted to provide `ReactElement | string`..
 
There is a type suitable for children. `ReactNode` https://stackoverflow.com/a/57253387
I think this is better!

Reply:
If we use ReactElement, it seems that we can't get a string. it would be better to change to `ReactNode`.

Reply:
Change it :)